trading_report/trade_log.txt

```diff
@@ -0,0 +1,8 @@
+SOL Trading at 2023-01-20T16:56:28.323Z
```

Comment:
remove this or will it just be overwritten all the time?

Reply:
Planning to reset the file in the github action